Makefile

```diff
@@ -19,18 +18,68 @@ IMG ?= $(IMAGE_TAG_BASE):$(IMG_TAG)
 # You can use it as an arg. (E.g make bundle-build BUNDLE_IMG=<some-registry>/<project-name-bundle>:<tag>)
 BUNDLE_IMG ?= $(IMAGE_TAG_BASE)-bundle:v$(VERSION)
 
+# default platform type
+ODH_PLATFORM_TYPE ?= OpenDataHub
+
+ifeq ($(ODH_PLATFORM_TYPE), OpenDataHub)
+	VERSION ?= 2.33.0
+	OPERATOR_NAMESPACE ?= opendatahub-operator-system
+	CHANNELS=fast
+	ROLE_NAME=controller-manager-role
+	BUNDLE_DIR ?= bundle
+	BUNDLE_DOCKERFILE_FILENAME=bundle.Dockerfile
+	OPERATOR_PACKAGE=opendatahub-operator
+	CONTROLLER_GEN_TAGS=--load-build-tags=odh
+	KUSTOMIZE_BASE=config/default
+	KUSTOMIZE_DIR=config/manifests
+	MANAGER_DIR=config/manager
+	GO_RUN_ARGS=-tags=odh
+	CRD_DIR=config/crd
+	RBAC_DIR=config/rbac
+else
+	VERSION ?= 2.23.0
+	OPERATOR_NAMESPACE ?= redhat-ods-operator
+	CHANNELS=alpha,stable,fast
+	DEFAULT_CHANNEL=stable
+	ROLE_NAME=rhods-operator-role
+	BUNDLE_DIR ?= bundle.rhoai
+	BUNDLE_DOCKERFILE_FILENAME=bundle.rhoai.Dockerfile
+	OPERATOR_PACKAGE=rhods-operator
+	CONTROLLER_GEN_TAGS=--load-build-tags=rhoai
+	KUSTOMIZE_BASE=config/default.rhoai
+	KUSTOMIZE_DIR=config/manifests.rhoai
+	MANAGER_DIR=config/manager.rhoai
+	GO_RUN_ARGS=-tags=rhoai
+	CRD_DIR=config/crd.rhoai
+	RBAC_DIR=config/rbac.rhoai
+endif
```

Comment:
why not struct it like
```
$GIT_ROOT
- odh-config
  - crd
  - default
  - kueue-config
  - manager
  - rbac
  - sample
  - webhook
- rhoai-config
    - crd
    - default
    - kueue-config
    - manager
    - monitroing  // to be removed once we do tno deploy old stack
    - partners // is this still needed or just remove it?
    - prometheus // to be removed once we do not deploy old stack
    - rbac
    - sample
    - webhook
```
then you only need a top variable odh-config vs rhoai-config and keep sub-folder the same